packages/lesmis/src/components/LocationHeader.js

```diff
@@ -143,9 +144,14 @@ export const LocationHeader = () => {
                 )}
               />
             ) : (
-              <IconButton startIcon={<GetApp />}>
-                <I18n t="dashboards.export" />
-              </IconButton>
+              <DashboardExportModal
```

Comment:
I think it would be good to have a loading state for the button because when it is clicked it takes a while before anything shows up. However if we are going to move the export to the server than it is ok for now without the loading state.

Reply:
Will put that in mind.